README.md

```diff
@@ -1,16 +1,36 @@
 # React Design Patterns
 
-This repository holds the source code for some [PX Blue's design patterns](https://pxblue.github.io/patterns), written in React. 
+This repository holds the source code for some [PX Blue's design patterns](https://pxblue.github.io/patterns), written in React. The code based is arranged so that you can easily copy and paste the code examples into your own projects.
 
-> NOTE: once this goes public, we will be able to provide deeplinks into each pattern using CodeSandbox.
+## Current Patterns
+
+| Pattern                                                                        | View on Code Sandbox                                                                                                                     |
```

Comment:
Might be nice to do this in three columns (it wasn't immediately clear to me what clicking the link on the pattern name would do):
```
| Pattern Name | Description | Interactive Demo |
| -------------- | ------------| ----------------- |
| Action List      | View Description | CodeSandbox |
```

Reply:
Or maybe we even have a brief description (1 sentence) of each pattern in the table